easybuild/easyconfigs/i/IMOD/IMOD-4.12.62-foss-2023a-CUDA-12.1.1.eb

```diff
@@ -0,0 +1,121 @@
+easyblock = 'ConfigureMake'
+
+name = 'IMOD'
+version = '4.12.62'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://bio3d.colorado.edu/imod/'
+description = """IMOD is a set of image processing, modeling and display
+programs used for tomographic reconstruction and for 3D reconstruction of EM
+serial sections and optical sections. The package contains tools for assembling
+and aligning data within multiple types and sizes of image stacks, viewing 3-D
+data from any orientation, and modeling and display of the image files. IMOD
+was developed primarily by David Mastronarde, Rick Gaudette, Sue Held, Jim
+Kremer, Quanren Xiong, and John Heumann at the University of Colorado."""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+toolchainopts = {'pic': True}  # openmp leads to segfault with xftoxg tool
+
+# download manually from mercurial repository and create source tarball:
+# hg clone --debug http://bio3d.colorado.edu/imod/nightlyBuilds/IMOD
+# get lunch 
+# cd IMOD
+# hg update -r IMOD_4-12-62
+# rm .hg* -rf
+# cd ..
+# find IMOD -print|sort| tar -czf IMOD-`cat IMOD/.version`.tar.gz -T -
```

Comment:
let's use `download_instructions` here, see for example #19884

Reply:
@boegel how to use EasyBuild's sources archive?
https://bio3d.colorado.edu/imod/COPYRIGHT.txt: "_... All portions of IMOD, except as noted below, are open source under the General Public License (GPL) version 2.0. ..._"